cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -1614,6 +1614,34 @@ var InstanceTypes = map[string]*InstanceType{
 		GPU:          0,
 		Architecture: "amd64",
 	},
+	"inf2.48xlarge": {
+		InstanceType: "inf2.48xlarge",
+		VCPU:         192,
+		MemoryMb:     786432,
+		GPU:          0,
+		Architecture: "amd64",
+	},
+	"inf2.24xlarge": {
+		InstanceType: "inf2.24xlarge",
+		VCPU:         96,
+		MemoryMb:     393216,
+		GPU:          0,
+		Architecture: "amd64",
+	},
+	"inf2.8xlarge": {
+		InstanceType: "inf2.8xlarge",
+		VCPU:         32,
+		MemoryMb:     131072,
+		GPU:          0,
+		Architecture: "amd64",
+	},
+	"inf2.xlarge": {
+		InstanceType: "inf2.xlarge",
+		VCPU:         4,
+		MemoryMb:     16384,
+		GPU:          0,
+		Architecture: "amd64",
+	},	
```

Comment:
Trailing bit of whitespace is what's causing the failing check here, if we get that fixed up we can get this merged.
```suggestion
	},
```